### PR TITLE
[kommander 1.1] fix: change grafana subchart location

### PR DESCRIPTION
**Pull request by juliangieseke**
**What type of PR is this?**

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
* change location of grafana chart to our charts repo due to the old repo being disabled soon.
```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Comment by juliangieseke** [comment]
lint failing is expected:
> ✖︎ kommander => (version: "0.8.46", path: "stable/kommander") > Chart version not ok. Needs a version bump!

**Comment by DanielMSchmidt** [comment]
@juliangieseke Why did you remove the kubecost chart bundle?

**Comment by juliangieseke** [comment]
hm, no idea, all I did was running `helm dep up`. looking…

**Comment by juliangieseke** [comment]
thats weird - github UI bug? 🤔 

its there:https://github.com/mesosphere/charts/tree/jg/komm-0.8/grafana-location/stable/kommander/charts

`git status` doesnt show any diff:
```
$ git status
On branch jg/komm-0.8/grafana-location
Your branch is up to date with 'origin/jg/komm-0.8/grafana-location'.

nothing to commit, working tree clean
```

also no error after `helm dep up`:
```
$ helm dep up
Hang tight while we grab the latest from your chart repositories...
...Unable to get an update from the "local" chart repository (http://127.0.0.1:8879/charts):
	Get "http://127.0.0.1:8879/charts/index.yaml": dial tcp 127.0.0.1:8879: connect: connection refused
...Unable to get an update from the "coreos" chart repository (https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/):
	Failed to fetch https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/index.yaml : 404 Not Found
...Successfully got an update from the "kubefed-charts" chart repository
...Successfully got an update from the "mesosphere-staging" chart repository
...Successfully got an update from the "kommander" chart repository
...Successfully got an update from the "mesosphere" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete.
Saving 7 charts
Downloading kubeaddons-catalog from repo https://mesosphere.github.io/charts/staging
Downloading kommander-federation from repo https://mesosphere.github.io/yakcl/charts
Downloading kommander-licensing from repo https://mesosphere.github.io/yakcl/charts
Downloading auto-provisioning from repo https://mesosphere.github.io/yakcl/charts
Downloading kommander-ui from repo https://mesosphere.github.io/kommander/charts
Downloading kubecost from repo https://mesosphere.github.io/charts/stable
Downloading grafana from repo https://mesosphere.github.io/charts/stable
Deleting outdated charts
```

**Comment by juliangieseke** [comment]
Oh wait, github UI doesnt say its removed, but smaller. seems like the kubecost chart has changed? 🤔

**Comment by juliangieseke** [comment]
`git log kubecost-0.1.12.tgz` on `gh-pages` branch:
```
commit d9f51c72aaf83a4ed8f3b38f210553542cc17715
Author: Dispatch CI <56653984+d2iq-dispatch@users.noreply.github.com>
Date:   Mon Nov 2 21:16:00 2020 +0000

    publish this branch

commit d730b38a31aca8cd61d807b86d4c6a4c11169f42
Author: Dispatch CI <56653984+d2iq-dispatch@users.noreply.github.com>
Date:   Fri Aug 21 00:17:37 2020 +0000

    ensure helm is installed where it's executed (#759)

commit e97db5a2100b371f1ceedee479d52f539b119a32
Author: Dispatch CI <56653984+d2iq-dispatch@users.noreply.github.com>
Date:   Thu Aug 6 02:02:28 2020 +0000

    kubecost: Upgrade base chart to v1.63.0 (#722)  * chore: Update kubecost to v1.63.0  * chore: Bump kubecost chart version
```

seems like the file was indeed changed - a couple of times 🤔

**Comment by juliangieseke** [comment]
#759 seems to have changed the helm version (which might change files), no idea where that last commit came from 🤷 
@joejulian @samvantran @faiq any idea? 

Overall, I dont think its necessairly a bad thing, but genereally speaking I find it somewhat concerned to mutate existing releases :(

**Comment by joejulian** [comment]
Depends on what you consider changing the version. The Dispatchfile had overridden the version we purposefully specified in the Makefile. We should have a single source of truth and it should be the Makefile so that there is consistency between what you can run locally, and what's run in CI.

**Comment by juliangieseke** [comment]
Yeah, true. It changed it _back_.

**Review by joejulian — APPROVED** [review]

File: Dispatchfile [hunk 1]
```diff
@@ -100,6 +100,6 @@ dind_task("test-helm3",
 )
 
 action(tasks=["test-helm2","test-helm3"], on=pullRequest())
-action(tasks=["publish"], on=push(branches=["master"]))
+action(tasks=["publish"], on=push(branches=["release/kommander-0.8.x"]))
```
Comment (juliangieseke) [inline Dispatchfile:103]: oh boy - good catch!

this shouldnt be hardcoded in the first place. is there any way to get the target branch from env vars? cc @faiq any idea?

**Pull request merged**